packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -4,225 +4,242 @@ import type { BadgeState } from './Badge.types';
 
 export const badgeClassName = 'fui-Badge';
 
-const useStyles = makeStyles({
-  root: {
+const useRootStyles = makeStyles({
+  base: {
     display: 'inline-flex',
     boxSizing: 'border-box',
     alignItems: 'center',
     justifyContent: 'center',
-    backgroundColor: tokens.colorBrandBackground,
-    ...shorthands.borderColor(tokens.colorBrandBackground),
-    color: tokens.colorNeutralForegroundOnBrand,
```

Comment:
Moved these into styles specific to the `brand` color. That way, the base style doesn't have any colors, and there is a unique style for every combination of `appearance` and `color`.